api-docs/swagger.yml

```diff
@@ -147,7 +147,7 @@ paths:
           description: Internal error with swarm or request JSON invalid
   /async-function/{functionName}:
     post:
-      summary: 'Invoke a function asynchronously in OpenFaaS - coming in PR 131'
+      summary: 'Invoke a function asynchronously in OpenFaaS'
```

Comment:
Do we need to call out the fact that this is only true if the stack is deployed in async mode - not the case by default right?

Reply:
Yes we do - but I think I'll merge because this is less wrong than saying about PR 131.